webapp/src/components/AssetPage/CategoryBadge/CategoryBadge.tsx

```diff
@@ -2,13 +2,32 @@ import React from 'react'
 import { t } from 'decentraland-dapps/dist/modules/translation/utils'
 import IconBadge from '../IconBadge'
 import { Props } from './CategoryBadge.types'
+import { getSearchWearableSection } from '../../../modules/routing/search'
+import { locations } from '../../../modules/routing/locations'
+
+const CategoryBadge = ({ wearable, assetType }: Props) => {
+
+  const section = getSearchWearableSection(wearable.category)
+
+  const href = React.useMemo(() => {
```

Comment:
We could just import `useMemo` right?

```suggestion
  useMemo(() => {
```